docs/pages/guides/nextjs-starter-kit.mdx

```diff
@@ -0,0 +1,545 @@
+export type User = {
+  // Your new property
+  randomNumber: number;
+  // ...
+};
+```
+
+Then make sure to return this new property in `/lib/server/database/getUser.ts`.
+
+```ts file="lib/server/database/getUser.ts"
+return { randomNumber: Math.random() /* ... */ };
+```
+
+The new property will now be available to use in your app:
+
+```ts
+// randomNumber: Math.random(),
+console.log(session.user.info.randomNumber);
+```
+
+#### Adding this to your Liveblocks app (optional)
+
+If you'd like your new property to be available inside your Liveblocks app (e.g.
+the React hooks used in whiteboard), you must modify `UserInfo` in
+`/liveblocks.config.ts`
+
+```ts file="liveblocks.config.ts"
+export type UserInfo = Pick<User, "randomNumber" /* ... */>;
+```
+
+Then modify `/pages/api/liveblocks/auth.ts`. First, we'll give an anonymous user
+a property:
+
+```ts file="pages/api/liveblocks/auth.ts"
+// Anonymous user info
+const anonymousUser: User = {
+  randomNumber: Math.random(),
+  // ...
+};
+```
+
+Next, we'll get the signed-in user's property:
+
+```ts file="pages/api/liveblocks/auth.ts"
+// Get current user info from session (defined in /pages/api/auth/[...nextauth].ts)
+// If no session found, this is a logged out/anonymous user
+const {
+  randomNumber,
+  // ...
+} = session?.user.info ?? anonymousUser;
+```
+
+And then pass this info to `authorize`:
+
+```ts file="pages/api/liveblocks/auth.ts"
+// Get Liveblocks access token
+const { data, error } = await authorize({
+  userInfo: { randomNumber /* ... */ },
+  // ...
+});
+```
+
+To make sure to check everything's hooked up correctly:
+
+```
+npm run typecheck
+```
+
+Once that's working, the new property can then be used in your app:
+
+```tsx
+// My random number
+const myRandomNumber = useSelf((me) => me.info.randomNumber);
+
+// An array of everyone else's random numbers
+const everyonesRandomNumbers = useOthersMapped(
+  (other) => other.info.randomNumber
+);
+```
+
+### Adding multiple authentication providers [#add-multiple-authentication-providers]
+
+It's possible to add multiple authentication providers to the starter kit using
+[NextAuth Providers](https://next-auth.js.org/providers/). Open
+`/pages/api/auth/[...nextAuth].ts` and place your providers in the object:
+
+```ts
+import GithubProvider from "next-auth/providers/github";
+import Auth0Provider from "next-auth/providers/auth0";
+
+export const authOptions = {
+  // ...
+  providers: {
+    GithubProvider({
+      clientId: process.env.GITHUB_CLIENT_ID as string,
+      clientSecret: process.env.GITHUB_CLIENT_SECRET as string,
+    }),
+    Auth0Provider({
+      clientId: process.env.AUTH0_CLIENT_ID as string,
+      clientSecret: process.env.AUTH0_CLIENT_SECRET as string,
+      issuer: process.env.AUTH0_ISSUER_BASE_URL as string,
+    }),
+  }
+};
+```
+
+It's not only possible with GitHub and Auth0, any
+[NextAuth provider](https://next-auth.js.org/providers/) will work.
+
+### Switching themes
```

Comment:
Have we considered incorporating a dark/light mode switch from the UI?

If not, could we explain what to modify in the CSS to force a theme? 

I’ve been doing that in my app "[LineCoach](https://github.com/adigau/line-coach-by-liveblocks)", and had to figure out what to change, but I clearly did something wrong cause when I deploy it, some friends see it in dark, and I see it light 🙈 I'll fix it later lol)